docs/source/configure/index.rst

```diff
@@ -18,39 +18,8 @@ at ``example/config.sample``
 We will start with looking at config file and then look at how Iroha can be configured with
 `environment parameters <#environment-variables>`_.
 
-.. code-block:: javascript
-  :linenos:
-
-  {
-    "block_store_path": "/tmp/block_store/",
-    "torii_port": 50051,
-    "torii_tls_params": {
-      "port": 55552,
-      "key_pair_path": "/path/to/the/keypair"
-    },
-    "internal_port": 10001,
-    "pg_opt": "host=localhost port=5432 user=postgres password=mysecretpassword dbname=iroha",
-    "database": {
-      "host": "localhost",
-      "port": 5432,
-      "user": "postgres",
-      "password": "mysecretpassword",
-      "working database": "iroha_data",
-      "maintenance database": "postgres"
-    },
-    "max_proposal_size": 10,
-    "proposal_delay": 5000,
-    "vote_delay": 5000,
-    "mst_enable" : false,
-    "mst_expiration_time" : 1440,
-    "max_rounds_delay": 3000,
-    "stale_stream_max_rounds": 2,
-    "utility_service": {
-      "ip": "127.0.0.1",
-      "port": 11001
-    },
-    "metrics":"127.0.0.1:8080"
-  }
+.. remoteliteralinclude:: https://raw.githubusercontent.com/hyperledger/iroha/main/example/config.sample
```

Comment:
Please use local file from current branch when you build documentation

Reply:
.

Reply:
@kuvaldini if only it was possible to do that. I'll try to find another solution - maybe it will be possible but I am not sure that that is the case.